.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,17 @@
+on: [push]
+name: "CI"
```

Comment:
Quotes required?

Reply:
This was actually merged with another PR. I'm hoping there wont be any issues when I merge. I'll revisit this once merged